packages/web/src/ProgressBar.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import { Box, createStyles, makeStyles, Theme } from '@material-ui/core';
+
+const useStyles = makeStyles((theme: Theme) =>
```

Comment:
O Theme existe para quando queremos variar alguma característica css por algum fator, como tema noite e tema dia, qual a utilidade dele aqui?

Reply:
Esse theme é usado igual o GRID.GET do ditouai, o padrao dele são 8px de espacamento no caso do spacing então usamos o theme.spacing pra padronizar o espacamento do codigo e tbm caso um dia for preciso mudar o espacamento pra 4 ou pra 12 ou pra qual for vai ser somente mudar no default e n vai precisar ficar passando em cada parte do codigo.